instruments/technical_docs_for_high_risk_ai.yaml

```diff
@@ -0,0 +1,349 @@
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ ]
+    requirements: [ ]
+    template: *systemcard_template
+  - question: "Beschrijf de monitoring, werking en controle van het AI-systeem, met betrekking tot: nauwkeurigheid voor specifieke en/of groepen personen en tov het beoogde doel, voorzienbare onbedoelde resultaten en risicobronnen voor de gezondheid."
+    urn: "urn:nl:aivt:ir:td:1.0:3"
+    labels:
+      - key: topic
+        value: "Toezicht en monitoring"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ ]
+    requirements: [ ]
+    template: *systemcard_template
+  - question: "Geef een beschrijving van de geschiktheid van de prestatiestatistieken voor het specifieke AI-systeem."
+    urn: "urn:nl:aivt:ir:td:1.0:4"
+    labels:
+      - key: topic
+        value: "Prestatiestatistieken"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ {"text": "model card:  metrics"}]
+    requirements: [ ]
+    template: *systemcard_template
+  - question: "Beschrijf het systeem voor risicobeheer (raadpleeg hiervoor artikel 9 van de AI Veroderning)."
+    urn: "urn:nl:aivt:ir:td:1.0:5"
+    labels:
+      - key: topic
+        value: "Risicobeheer"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ {"text": "system card:  risk_management"} ]
+    requirements: [ ]
+    template: *systemcard_template
+  - question: "Beschrijf de relevante wijzigingen die door de aanbieder tijdens de levensduur van het systeem worden aangebracht."
+    urn: "urn:nl:aivt:ir:td:1.0:6"
+    labels:
+      - key: topic
+        value: "Systeemaanpassingen"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ ]
+    requirements: [ ]
+    template: *systemcard_template
+  - question: "Geef een lijst van de geharmoniseerde normen die volledig of gedeeltelijk worden toegepast en waarvan de referenties zijn gepubliceerd in het Publicatieblad van de Europese Unie. Indien de geharmoniseerde normen niet zijn toegepast, geef dan een gedetailleerde beschrijving van de vastgestelde oplossingen om te voldoen aan de vereisten van hoog-risico AI-systemen, met inbegrip van een lijst van andere toegepaste relevante normen en technische specificaties." #to do: shorten it one and include link?
+    urn: "urn:nl:aivt:ir:td:1.0:7"
+    labels:
+      - key: topic
+        value: "Geharmoniseerde normen lijst"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ ]
+    requirements: [ ]
+    template: *systemcard_template
+  - question: "Onderteken een exemplaar van de EU-conformiteitsverklaring (raadpleeg hiervoor artikel 47 en bijlage 5 van de AI Verordening)."
+    urn: "urn:nl:aivt:ir:td:1.0:8"
+    labels:
+      - key: topic
+        value: "EU-conformiteitsverklaring"
+    type: ["form_filling", "upload_document"]
```

Comment:
I was expecting the form itself here as well (for form_filling tasks).

Reply:
Understandable! Unfortunately there is not 1 format. So we can decide to make one, or leave it up to the user.